### DISCUSSION TITLE
Very hard to dual wield a storage container and another item

### DISCUSSION THREAD

**Discussion by ExcavatorMann** [FakeFishGames/Barotrauma#11166] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I picked up a storage container and tried to equip a revolver from my inventory in the other hand, but the revolver got put into the storage container. It seems that the only way to equip an item in the other hand while holding a storage container is to dual wield items before picking up the storage container or pick up the item after the storage container is equipped.

### Reproduction steps

1. Pick up a storage container (with or without an item in the right hand)
2. Try to equip a one-handed weapon/tool from your inventory
3. The one-handed weapon/tool gets sucked into the storage container

### Bug prevalence

Happens every time I play

### Version

0.20.16.1

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
I wouldn't consider this a bug: equipping any item always puts it inside the held item if that's possible. In my opinion adding an exception to that rule for some specific items seems like it'd make the inventory behavior more inconsistent and complex. To me it's not immediately obvious which items the exception should apply to: handheld containers that don't do anything with the contained items (excluding things like tools and guns)? Handheld items that do something, excluding things like materials, fuel rods - but what about things like syringes or throwable items? So personally I'm a little hesitant to change the behavior just to address a rare use-case like this (especially since there _are_ ways to equip both the revolver and the container at the same time even with the current inventory behavior).
